ScoutSuite/providers/aws/resources/ec2/instances.py

```diff
@@ -28,14 +28,15 @@ def __init__(self, facade: AWSFacade, region: str, vpc: str):
 
         get_name(raw_instance, instance, 'InstanceId')
         get_keys(raw_instance, instance,
-                 ['KeyName', 'LaunchTime', 'InstanceType', 'State', 'IamInstanceProfile', 'SubnetId'])
+                 ['KeyName', 'LaunchTime', 'InstanceType', 'State', 'IamInstanceProfile', 'SubnetId', 'TagSet'])
```

Comment:
I added this field to include the TagSet.  I probably didn't include the flat-mapping of TagSet to Tags mapping, but I can do that.  We have a downstream ETL process to do some light massaging of data, so that's probably where I put it.

Reply:
Sure, all I know is that I added tags to an EC2 instance but they didn't show up in the report. If you're happy with the implementation I have no issue with merging.

Reply:
I'm having a lot of issues with the case-sensitivity of the branches, so I'm deleting this one and re-creating a PR witht he `Enhancements/` folder..